telegram/ext/_commandhandler.py

```diff
@@ -98,12 +103,13 @@ def __init__(
         super().__init__(callback, block=block)
 
         if isinstance(command, str):
-            self.command = [command.lower()]
+            commands = frozenset((command.lower(),))
         else:
-            self.command = [x.lower() for x in command]
-        for comm in self.command:
+            commands = frozenset(x.lower() for x in command)
+        for comm in command:
```

Comment:
if `command` is a string, this will iterate through the string, which is wrong

Reply:
I really shouldn't try to finish a PR while in a hurry ... Sorry, I thought I had already changed that

Reply:
Turns out I forgot to push a commit …